profile/CSIP.xml

```diff
@@ -166,8 +166,8 @@
             </requirement>
             <requirement ID="CSIP2" REQLEVEL="MUST" RELATEDMAT="VocabularyType" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
                 <description>
-                    <head>General content type</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">The @type attribute must be used to provide a general description of the content information type of the package. A fixed vocabulary is used which will evolve under the curation of the DILCIS Board as additional content information type specifications are developed.</p>
+                    <head>Content type</head>
+                    <p xmlns="http://www.w3.org/1999/xhtml">The @TYPE attribute must be used to declare the content information type of the package. Legal values are defined in a fixed vocabulary. When the content type used falls outside of the defined vocabulary the @TYPE value must be set to "OTHER". The vocabulary will evolve under the curation of the DILCIS Board as additional content information type specifications are developed.</p>
```

Comment:
```suggestion
                    <p xmlns="http://www.w3.org/1999/xhtml">The @TYPE attribute MUST be used to declare the type of content held in the package, e.g. book, journal, stereograph, dataset, video, etc. (legal values are defined in a fixed vocabulary). When the content type used falls outside of the defined vocabulary, the @TYPE value must be set to "OTHER" and the specific value declared in @csip:OTHERTYPE. The vocabulary will evolve under the curation of the DILCIS Board.</p>
```

- I borrowed the list of examples from mets.xsd, aiming to give the reader some broad understanding of what kind of "type" is being discussed, without the need to look up the actual values from the vocabulary
- a standardised term for payload content would be welcome here: "content information," "content files," "payload data" etc. Consistent use would make it easier, as pretty much any term requires explanation, e.g. "content files" can be seen to cover both data and metadata, as both are the IP's content
- shortened the last sentence as `@TYPE` is not related to content information type specifications